app/views/text_components/_form.html.haml

```diff
@@ -3,14 +3,11 @@
 
   .form-inputs
 
-    = f.association :report, selected: @report.id
-
```

Comment:
Are you sure we should remove that field? I think @drjakob and his team are depending on it to move text components between reports…